boards/sc/scobc_v1/board.cmake

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2025 Space Cubics Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include(${ZEPHYR_BASE}/boards/common/xsdb.board.cmake)
+set(SUPPORTED_EMU_PLATFORMS qemu)
+set(QEMU_ARCH xilinx-aarch64)
+set(QEMU_CPU_TYPE_${ARCH} cortex-a78ae)
+
+set(QEMU_FLAGS_${ARCH}
```

Comment:
There is no QEMU dts so all these changes are not required or add qemu dts and compile it